README.md

```diff
@@ -95,6 +96,18 @@ If you're using a non-version specific tag (`stable` or `latest`) or if the vers
 
 If you're using a version-specific tag and the new version of the domain server you want to upgrade to doesn't match the tag you use, you need to first change the tag you use and then restart your container. (`v0` matches any v0.x.x version, `v0.5` matches any v0.5.x version, and so on.)
 
+## Generating random passwords
```

Comment:
Should this section be in a separate file?

Reply:
I think it's good if it's directly in the readme.